packages/safe-ds-lang/tests/resources/generation/eject/output/tests/generator/eject/gen_input.py

```diff
@@ -0,0 +1,8 @@
+# Pipelines --------------------------------------------------------------------
+
+def test():
+    f((g() or g()))
+    f((g() and g()))
```

Comment:
These are not equivalent to the runner calls, since the right operand of `or`/`and` no longer get executed even if the left operand determines the result. The Safe-DS `or`/`and` is deliberately not short-circuited.

You could instead generate another immediately-invoked lambda.

Reply:
Another option would be to add the functions that are currently in `runner.codegen` to the generated code and call those. This would be more readable than the lambda, IMO, since the function has a proper name. Since this feature is used when users want to edit the Python code afterward, the generated code should be as readable as possible.

I'd imagine something like this:

```py
# Utils ------------------------------------------------------------------------

def __gen_eager_or(left_operand: bool, right_operand: bool) -> bool:
    return left_operand or right_operand


def __gen_eager_and(left_operand: bool, right_operand: bool) -> bool:
    return left_operand and right_operand

# Pipelines --------------------------------------------------------------------

def test():
    f(__gen_eager_or(g(), g()))
    f(__gen_eager_and(g(), g()))
```

The utils should only be added when they are needed. We can then always generate this code, regardless of the `disableRunnerIntegration` flag and remove the `src/safeds_runner/codegen/_operators.py` file in the runner.

In that case, the difference between the two generation modes is simply whether calls get memoized and placeholders get saved.